doc/tor.1.txt

```diff
@@ -2605,6 +2605,19 @@ Relays publish most statistics in a document called the
 extra-info document. The following options affect the different
 types of statistics that Tor relays collect and publish:
 
+[[BandwidthStatistics]] **BandwidthStatistics** **0**|**1**|**auto**::
+    Relays and bridges only. 
+	When this option is enabled or left unspecified, Tor collects
+    statistics about bandwidth used each 24 hours for the
+    last 5 days (if available) and writes them to disk every 24 hours.
+    Enables relay and bridge operators to monitor bandwidth used by Tor.
+    Tor includes the bandwidth statistcs in extrainfo document if and
+    only if BandwidthStatistics is enabled or left unspecified,
+    in which case the consensus parameter, BandwidthStatistics,
+    is checked further whether it is enabled or not.
+    If ExtraInfoStatistics is enabled, it will published as part of
+    extra-info document. (Default: auto)
```

Comment:
We've tried revising this man page entry a few times already. So I'm going to show you how to create a man page entry, using another man page entry as a template.

Take a copy of a similar man page entry. Let's use CellStatistics:
```
[[CellStatistics]] **CellStatistics** **0**|**1**::
    Relays only.
    When this option is enabled, Tor collects statistics about cell
    processing (i.e. mean time a cell is spending in a queue, mean
    number of cells in a queue and mean number of processed cells per
    circuit) and writes them into disk every 24 hours. Onion router
    operators may use the statistics for performance monitoring.
    If ExtraInfoStatistics is enabled, it will published as part of
    extra-info document. (Default: 0)
```

Change the copy so that it documents the new option. Only make the changes that are needed:
```
[[BandwidthStatistics]] **BandwidthStatistics** **0**|**1**|**auto**::
    Relays and bridges only.
    When this option is enabled, Tor collects statistics about bandwidth
    (i.e. read and written bytes, and read and written directory bytes) and
    writes them into disk every 24 hours. Relay and bridge
    operators may use the statistics for performance monitoring.
    If ExtraInfoStatistics is enabled, it will published as part of
    the extra-info document. (Default: auto)
```

This option also has a consensus parameter. So we also need to find a man page entry that documents a consensus parameter. Let's use the consensus parameter sentence from NumDirectoryGuards:
```
    If this option is set to 0, use the
    value from the guard-n-primary-dir-guards-to-use consensus parameter, and
    default to 3 if the consensus parameter isn't set. 
```

Change the copy so that it documents the new consensus parameter. Only make the changes that are needed:
```
    If this option is set to "auto", use the value from the BandwidthStatistics
    consensus parameter, and default to 1 if the consensus parameter isn't set.
```

Now, combine the two parts into a man page entry, and wrap the lines:
```suggestion
[[BandwidthStatistics]] **BandwidthStatistics** **0**|**1**|**auto**::
    Relays and bridges only.
    When this option is enabled, Tor collects statistics about bandwidth (i.e.
    read and written bytes, and read and written directory bytes) and writes
    them to disk every 24 hours. Relay and bridge operators may use the
    statistics for performance monitoring. If this option is set to "auto",
    use the value from the BandwidthStatistics consensus parameter, and
    default to 1 if the consensus parameter isn't set. If ExtraInfoStatistics
    is enabled, it will published as part of the extra-info document.
    (Default: auto)
```

Sometimes, fewer changes are better.